### DISCUSSION TITLE
安装1panel之后为什么检测不到服务器上的网站与应用？

### DISCUSSION THREAD

**Discussion by shijili** [1Panel-dev/1Panel#4160] [Q&A]
我的服务器之前已经安装了mariadb、nginx、wordpress、php，然后安装了1panel，为什么检测不到我服务器上的应用和网站？万分感谢！

**Comment by wanghe-fit2cloud** [comment]
已安装应用列表检测不到手动安装的容器服务，需要进入 `容器` 列表查看、管理。
> 应用和网站都需要通过 1Panel 来创建。
